src/backend/joanie/core/models/events.py

```diff
@@ -31,6 +69,17 @@ class Event(BaseModel):
         default=enums.EVENT_INFO,
     )
     read = models.BooleanField(_("read"), default=False)
+    context = EventContextField(
+        _("context"),
+        default=dict,
+        blank=True,
+    )
+    type = models.CharField(
+        _("type"),
+        max_length=20,
+        choices=enums.EVENT_TYPE_CHOICES,
+        default=enums.EVENT_TYPE_NOTIFICATION,
+    )
```

Comment:
In order to prevent to generate a migration each time we update the `EVENT_TYPE_CHOICES` tuple, we could make this choice list lazy.